### ISSUE TITLE
Application window larger than screen

### ISSUE THREAD

**Issue by farsighted6** [SashaCX75/AmazFit_Watchface_Editor_2#6]
When I run AmazFit WatchFace Editor 2, half of the window is outside of my screen. The maximize button is missing (grayed out, disabled, doesn't work). The window has a fixed default size and can not be resized. Since the fixed default size is larger than my screen size, AmazFit WatchFace Editor 2 is unusable. The fonts are blurry because "DPI awareness" for high DPI is not enabled in AmazFit WatchFace Editor 2.

I'm using Windows 10, resolution 1920x1080 with 175 DPI on a 52 inch screen.

Can you make the following changements to solve this bug : 
- add a window maximize button that is enabled (= not grayed out)
- make the window resizable
- make the default size of the window smaller
- open your application as maximized
- enable "DPI awareness" (needed for high DPI to avoid blurry fonts) and add an option in your application so that users can change this setting

See here a screenshot of my Windows 10 screen when starting AmazFit WatchFace Editor 2 : 

![screen](https://user-images.githubusercontent.com/97792802/149624488-94f51a8b-87d2-44c7-9004-fb3d06e8c2b6.png)

**Comment by SashaCX75** [comment]
I did not plan to add any of the above to the editor. Now the project is no longer supported, and even more so, I will not make any changes. I can turn on "DPI awareness" which will remove blurry fonts, but other distortions may appear. If you want, you can see the result yourself.
[AmazFit_Watchface_Editor_2_v7.3_(DPI_awareness).zip](https://github.com/SashaCX75/AmazFit_Watchface_Editor_2/files/7877805/AmazFit_Watchface_Editor_2_v7.3_.DPI_awareness.zip)

**Comment by farsighted6** [comment]
Thanks for the AmazFit_Watchface_Editor_2_v7.3_(DPI_awareness).zip. This removed the blurry fonts.

I also have the same issue with the project "AmazFit-Watch-Faces-Editor-Type-GT3". The window of "AmazFit-Watch-Faces-Editor-Type-GT3" has a fixed size, window is larger than screen, no maximize button and window is not resizable. Since both projects are using the same GUI, maybe you can rethink and solve the GUI problem for both projects ?

**Comment by SashaCX75** [comment]
No. I don't have time to figure out how to properly scale the screen to different proportions and then redo the interface. And then check whether the inscription in one of the languages has been cut off.
